apps/api/src/app/layouts-v2/usecases/get-layout/get-layout.command.ts

```diff
@@ -1,8 +1,12 @@
-import { IsString, IsDefined } from 'class-validator';
-import { EnvironmentWithUserObjectCommand } from '@novu/application-generic';
+import { IsString, IsBoolean, IsOptional } from 'class-validator';
+import { EnvironmentWithUserCommand } from '@novu/application-generic';
 
-export class GetLayoutCommand extends EnvironmentWithUserObjectCommand {
+export class GetLayoutCommand extends EnvironmentWithUserCommand {
   @IsString()
-  @IsDefined()
-  layoutIdOrInternalId: string;
+  @IsOptional()
+  layoutIdOrInternalId?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  skipAdditionalFields?: boolean;
```

Comment:
The flag allows only the return of essential information about the layout, without the schema and controls. Used in other use cases, for example, `duplicate`, where we don't need this info.